lldb/source/Core/CMakeLists.txt

```diff
@@ -71,8 +71,6 @@ add_lldb_library(lldbCore
     lldbUtility
     lldbValueObject
     lldbVersion
-    lldbPluginCPlusPlusLanguage
-    lldbPluginObjCLanguage
```

Comment:
You can now add `NO_PLUGIN_DEPENDENCIES` to `add_lldb_library` above and resolve the FIXME. 🥳